Orange/base.py

```diff
@@ -69,7 +69,11 @@ def preprocess(self, data):
         return data
 
     def __repr__(self):
-        return self.name
+        return '{}({})'.format(type(self).__name__,
+            ", ".join("{}={}".format(k, repr(v))
+                for k, v in self.params.items()) if
+                "params" in dir(self) else ""
+       )
```

Comment:
Can you make `Learner` and `Model` work with `Reprable` too? For the moment, checking `getattr(self, arg)` or `self.params[arg]` should do the job.